locale/en/get-involved/node-meetups.md

```diff
@@ -343,6 +343,14 @@ REQUIREMENTS
 
 #### California
 
+#### Los Angeles
+
+- [js.la](https://js.la)
+- Frequency of meetups - monthly
+- How to submit a talk? [contribute.js.la](https://contribute.js.la)
+- Organizer name - David Guttman
+- Organizer contact info - 
```

Comment:
So are you sure no contact info? I'm OK with other info :)

Reply:
Ok, I added my slack info which is the best way to reach me =)